common.yml

```diff
@@ -3,11 +3,6 @@
   sudo: yes
   roles:
     - bennojoy.redis
-
-- hosts: "{{ hosts_prefix }}-tsuru-db"
-  sudo: yes
-  sudo_user: root
-  roles:
```

Comment:
The commit message doesn't mention why this changes. Which probably also means that it should be in a separate commit.